legal-api/migrations/versions/957a67bde783_add_in_liquidation_column_to_businesses.py

```diff
@@ -0,0 +1,29 @@
+"""Add in_liquidation column to businesses
+
+Revision ID: 957a67bde783
+Revises: 1ec937c2466e
+Create Date: 2025-05-29 11:43:23.869675
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '957a67bde783'
+down_revision = '1ec937c2466e'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('businesses', sa.Column('in_liquidation', sa.Boolean(), nullable=True))
+    op.add_column('businesses_version', sa.Column('in_liquidation', sa.Boolean(), autoincrement=False, nullable=True))
```

Comment:
You can add default value for it. Ref: https://github.com/bcgov/lear/blob/main/legal-api/migrations/versions/8bd6dc383c96_add_no_dissolution_to_businesses.py#L20-L21